packages/front/svelte/views/not-found/NotFound.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+    import NotFoundController from "./NotFound.controller";
+    import Button from "@dsfr/Button.svelte";
+
+    const ctrl = new NotFoundController();
+</script>
+
+<div class="flex">
+    <div class="same-size">
+        <h1>Page non trouvée</h1>
+        <p class="fr-text--sm">Erreur 404</p>
+        <p class="fr-text--lead">La page que vous cherchez est introuvable. Excusez-nous pour la gêne occasionnée.</p>
+        <p>
+            Si vous êtes arrivé sur cette page en tapant une url depuis votre navigateur, c’est probable que vous vous
+            soyez trompé d’url.
+        </p>
+        <p>
+            Si vous êtes arrivé sur cette page en cliquant sur un lien du site, merci de nous contacter via
+            <u>contact@datasubvention.bata.gouv.fr</u>
```

Comment:
```suggestion
            <u>contact@datasubvention.beta.gouv.fr</u>
```
J'ai vu que la typo était sur le design mais c'est quand même très vraisemblablement une typo donc je pense qu'on peut le corriger

Reply:
bien vu